toolchain/Config.in

```diff
@@ -238,6 +238,7 @@ comment "C Library"
 choice
 	prompt "C Library implementation" if TOOLCHAINOPTS
 	default LIBC_USE_UCLIBC if arc
+	default LIBC_USE_GLIBC if powerpc64
 	default LIBC_USE_MUSL
```

Comment:
It does, but it only implements the v2 ABI. I haven't managed to get a working build of LEDE using that ABI in big-endian mode yet.